package-lock.json

```diff
@@ -0,0 +1,4549 @@
+{
```

Comment:
We should ignore this file as others SPAs?

Reply:
Oops. Nice catch. I didn't notice this.